DESCRIPTION

```diff
@@ -15,6 +15,16 @@ Authors@R:
            family = "Funk",
            role = "aut",
            email = "sebastian.funk@lshtm.ac.uk"),
+    person(given = "Carl",
+           family = "Pearson",
+           role = "aut",
+           email = "carl.ab.pearson@gmail.com",
+           comment = c(ORCID = "0000-0003-0701-7860")),
+    person(given = "Hugo",
+           family = "Gruson",
+           role = c("aut"),
+           email = "hugo.gruson@lshtm.ac.uk",
```

Comment:
```suggestion
           email = "hugo.gruson@lshtm.ac.uk",
```

This address was deleted the second I left LSHTM. Could you please replace it by hugo.gruson@normalesup.org or just remove it?